test/function.civet

```diff
@@ -270,9 +270,7 @@ describe "function", ->
       ---
       (@a, @b) ->
       ---
-      (function(a, b) {this.a = a;
-      return this.b = b;
-      })
+      (function(a, b) {this.a = a;return this.b = b;})
```

Comment:
This was a bug before but I don't think `b` (or anything) should be returned in this function.